slack-java-client/src/main/java/com/hubspot/slack/client/ratelimiting/ByMethodRateLimiter.java

```diff
@@ -0,0 +1,20 @@
+package com.hubspot.slack.client.ratelimiting;
+
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+import com.google.common.util.concurrent.RateLimiter;
+import com.hubspot.slack.client.methods.SlackMethod;
+
+public class ByMethodRateLimiter implements SlackRateLimiter {
+  private static final ConcurrentMap<SlackMethod, RateLimiter> RATE_LIMITERS = new ConcurrentHashMap<>();
+
```

Comment:
You need to add an `@Inject` constructor here, otherwise guice doesn't know how to put this all together.